build.sbt

```diff
@@ -3,12 +3,11 @@ lazy val `play-webgoat` = (project in file(".")).enablePlugins(PlayScala)
 name := "play-webgoat"
 version := "1.0"
 
-crossScalaVersions := Seq("2.13.10")
+crossScalaVersions := Seq("2.13.10", "3.3.0-RC3")
 scalaVersion := crossScalaVersions.value.head // tc-skip
 
 libraryDependencies ++= Seq(guice, ws)
 scalacOptions ++= Seq(
-  "-feature", "-unchecked", "-deprecation", "-Xfatal-warnings",
-  // "unused" is too fragile w/ Twirl, routes file
-  "-Xlint:-unused"
+  // "-unchecked", "-deprecation" // Set by Play already
+  "-feature", "-Werror",
```

Comment:
@SethTisue The `"-Xlint:-unused"` was doing nothing in Scala 2, so I just removed it (and because Scala 3 doesn't recognize it). I think what you actually wanted was `"-Xlint:-unused,_"` which disables the unused flag but enables the rest...
See 
* https://github.com/scala/bug/issues/10294#issuecomment-298168471
* https://github.com/scala/bug/issues/10270#issuecomment-298262073

Feel free to open a PR with a replacement that works in Scala 2 + 3. I am not going to do that right now. Removing it is not changing the status quo :wink:

Reply:
#95